src/helpers.ts

```diff
@@ -308,3 +308,19 @@ export const combineSDJWT = (jwt: string, disclosures: string[], kbjwt?: string)
 
   return combined;
 };
+
+export const createDecoy = (count: number, hasher: Hasher): string[] => {
+  if (count < 0) {
+    throw new CreateDecoyError('decoy count must not be less than zero');
+  }
+
+  const decoys = [];
+
+  for (let i = 0; i < count; i++) {
+    const salt = generateSalt(16);
```

Comment:
@ragnika where does `generateSalt` come from? Didn't we make it configurable?

Reply:
@linasi Yes, it's configurable but there's a default one in the helper.

This is using that default one.

Reply:
> @linasi Yes, it's configurable but there's a default one in the helper.
> 
> This is using that default one.

I understand, but if I pass my custom implementation it will not be used here, right?

Reply:
No, it won't.

will need a small rework.
I can address that in a future PR